packages/quantic/force-app/main/default/lwc/quanticSort/e2e/quanticSort.e2e.ts

```diff
@@ -0,0 +1,75 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+
+const newestDateSort = 'date descending';
+const defaultSortLabels = ['Relevancy', 'Newest', 'Oldest'];
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test;
+  if (useCase.value === useCaseEnum.search) {
+    test = fixtures[useCase.value] as typeof testSearch;
+  } else {
+    test = fixtures[useCase.value] as typeof testInsight;
+  }
+
+  test.describe(`quantic sort  ${useCase.label}`, () => {
+    test.describe(`when changing sort option to Newest`, () => {
+      test('should trigger a new search and log analytics', async ({
+        sort,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await sort.clickSortDropDown();
+        await sort.clickSortButton('Newest');
+        const searchResponse = await searchResponsePromise;
+        const {sortCriteria} = searchResponse.request().postDataJSON();
+        expect(sortCriteria).toBe(newestDateSort);
+        await sort.waitForSortUaAnalytics(newestDateSort);
+      });
+    });
+
+    test.describe('when testing accessibility', () => {
+      test('should be accessible to keyboard', async ({sort}) => {
+        let selectedSortLabel = await sort.sortDropDown.textContent();
+        expect(selectedSortLabel).toEqual(defaultSortLabels[0]);
+
+        // Selecting the next sort using Enter to open dropdown, the ArrowDown, then ENTER key
+        await sort.focusSortDropDown();
+        await sort.openSortDropdownUsingEnter();
+        await sort.selectSortOptionUsingArrow();
+        selectedSortLabel = await sort.sortDropDown.textContent();
+        expect(selectedSortLabel).toEqual(defaultSortLabels[1]);
```

Comment:
Again here, I would go fetch what the loaded sorts are available on the page instead of relying on the default sorts.

Reply:
same as above, we want to be sure the sort displays in correct order

Reply:
Order of sorts is not in the contract of our component. It's a parameter we receive and then display. We have unit tests to check that the sort are displayed in the order we've received them.

The e2e test shouldn't rely on the order of sort options to pass or fail. Please make this small change :)